packages/core/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.22.2",
+  "version": "2.22.3",
```

```suggestion
  "version": "2.22.3-alpha.1",
```

This version will also need to be updated in packages that use `@web3-onboard/core`, such as react, vue, solid. Please update the core version within those and bump those package versions with an alpha flag as shown above